client/windows/CCreatureWindow.cpp

```diff
@@ -521,8 +532,16 @@ void CStackWindow::CWindowSection::createCommanderAbilities()
 	}
 }
 
-void CStackWindow::setSelection(si32 newSkill, CClickableObject * newIcon)
+void CStackWindow::setSelection(si32 newSkill, CCommanderSkillIcon * newIcon)
 {
+	auto getSkillDescription = [this](int skillIndex, bool selected) -> std::string
+	{

```

Comment:
getSkillDescription actually uses "info" field from CStackWindow. There is also getSkillImage that is currently implemented as code duplicate in CStackWindow::CWindowSection::createCommander and void CStackWindow::setSelection so I follow existing design.